operator/pkg/ciliumendpointslice/controller.go

```diff
@@ -97,14 +118,23 @@ func registerController(p params) error {
 		ciliumEndpoint:      p.CiliumEndpoint,
 		ciliumEndpointSlice: p.CiliumEndpointSlice,
 		ciliumNodes:         p.CiliumNodes,
+		namespace:           p.Namespace,
 		slicingMode:         p.Cfg.CESSlicingMode,
 		maxCEPsInCES:        p.Cfg.CESMaxCEPsInCES,
 		rateLimit:           rateLimitConfig,
 		enqueuedAt:          make(map[CESName]time.Time),
 		metrics:             p.Metrics,
+		defaultCESSyncTime:  DefaultCESSyncTime,
+		priorityNamespaces:  make(map[string]struct{}),
+		Job:                 p.Job,
 	}
-
+	cesController.cond = *sync.NewCond(&lock.Mutex{})
```

Comment:
nit: `cond` field can be populated just above in the initialization of the `cesController` variable, like other fields.